src/__tests__/ParserComponent.test.tsx

```diff
@@ -84,9 +85,47 @@ describe('<ParserComponent /> Unit Tests', () => {
     });
   });
 
+  describe('createCountingObjects()', () => {
+    const pi: ParserInterface = {prompt: 'test', fileType: FileType.csv};
+    const pc = new ParserComponent(pi);
+    it('instantiates a list of CountTypes', () => {
+      const listObjects = pc.createTypeCountingObjects(4);
+    });
+  });
+
   describe('inferTypes()', () => {
-    it('dummy test', () => {
-      // todo: devs need to write unit tests
+    const pi: ParserInterface = {prompt: 'test', fileType: FileType.csv};
+    const pc = new ParserComponent(pi);
+    // eslint-disable-next-line no-array-constructor
+    const data = new Array(2);
+    data[0] = {money: 100, heart_attacks: '2016-07-03', animals: 'dog'};
+    data[1] = {money: 55, heart_attacks: '2019-02-02', animals: 'cat'};
```

Comment:
Similarly, all these variables should likely be initialized in `beforeEach` or `beforeAll`